src/Tribe/Admin/Notices.php

```diff
@@ -237,4 +238,36 @@ public function maybe_display_plus_commerce_notice() {
 			tribe_notice( "event-tickets-plus-missing-{$provider}-support", $message, 'dismiss=1&type=warning' );
 		}
 	}
+
+	/**
+	 * Display Settings help banner on Event Ticket's Settings.
+	 *
+	 * @since TBD
+	 */
+	public function maybe_display_ticket_settings_help_banner() {
```

Comment:
We should use tickets in plural so it's more obvious that's the tickets plugin settings and not a particular ticket.

```suggestion
	public function maybe_display_tickets_settings_help_banner() {
```